### PR TITLE
Small changes to the debug file and the logging variables

### PR DESCRIPTION
**Pull request by 10eMyrT**
This PR considers the comments made by @Tjzabel and made changes to the `debug.go` file as well as the names of the debug variables in other functions.

### CODE REVIEW

File: cmd/teleirc.go [hunk 1]
```diff
@@ -24,36 +24,35 @@ var (
...
-		verbose.LogError(err)
+		debug.LogError(err)
 		os.Exit(1)
 	}
 
 	var tgapi *tgbotapi.BotAPI
-	tgClient := tg.NewClient(settings.Telegram, tgapi, verbose)
-	tgChan := make(chan error)
+	tgClient := tg.NewClient(settings.Telegram, tgapi, debug)
+	tgChan   := make(chan error)
```
Comment (Tjzabel) [inline cmd/teleirc.go]: The `:=` on line 45 doesn't need to be in line with line 44. Padding is typically stylistically done when there are 3-4+ variables all being instantiated at the same time.

File: cmd/teleirc.go [hunk 2]
```diff
@@ -24,36 +24,35 @@ var (
...
 
-	ircClient := irc.NewClient(settings.IRC, verbose)
-	ircChan := make(chan error)
+	ircClient := irc.NewClient(settings.IRC, debug)
+	ircChan   := make(chan error)
```
Comment (Tjzabel) [inline cmd/teleirc.go]: Same here. Doesn't need to be in line.

**Comment by 10eMyrT** [comment]
The feedback from @Tjzabel has been addressed. Thank you @Tjzabel and @jwflory for the review. We can merge then.

**Comment by jwflory** [comment]
@10eMyrT Super, thanks for updating. In it goes! :soccer: :arrow_right: :goal_net:

**Pull request merged**